fearless/Common/Repository/ChainLocalRepository.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import RobinHood
+
+typealias ChainModelList = [ChainModel]
+
+extension ChainModelList: Identifiable {
+    public var identifier: String {
```

Comment:
why not let?

Reply:
Because we need extension to confirm protocol for typealias and swift don't allow extensions contain stored properties.